modules/shortcodes/gist.php

```diff
@@ -58,6 +58,7 @@ function jetpack_gist_get_shortcode_id( $gist = '' ) {
 	$gist_info = array(
 		'id'   => '',
 		'file' => '',
+		'ts'   => 8,
```

Comment:
I noticed you set the default to 8 here, and to 4 in `github_gist_simple_embed`. What do you think about having the same default everywhere? 4 seems like a sensible choice as 8 may push content a bit too much on sites with a narrow content width.

Reply:
I meant to be consistent, can adjust the function below. I chose 8 as that's the GitHub gist default.

Reply:
Fixed in `55d9440abcade22d64e60b490889c8641500c4e6`